modelstore/storage/blob_storage.py

```diff
@@ -87,7 +87,7 @@ def _get_metadata_path(
             model_id (str): A UUID4 string that identifies this specific
             model.
         """
-        versions_path = get_versions_path(domain, state_name)
+        versions_path = get_versions_path(self.root_dir, domain, state_name)
```

Comment:
This relies on all of the child classes of `BlobStorage` needing to have `root_dir`. Probably, in this case, it's worth putting `root_dir` (or a similarly named variable) here in this class, and having the child classes set it in their constructor with `super()`.

Reply:
that would be a good way to handle it.

Reply:
Working on this here: https://github.com/operatorai/modelstore/pull/116